tests/system/src/test/java/test/robot/com/sun/glass/ui/monocle/MonocleApplicationTest.java

```diff
@@ -140,7 +140,7 @@ public void testCharToKeyCode() throws Exception {
             for (int i = 0; i < TEST_CASES.length; i++) {
                 char ch = (char) TEST_CASES[i][0];
                 int expectedCode = TEST_CASES[i][1];
-                int code = MonocleApplicationShim._getKeyCodeForChar(ch);
+                int code = MonocleApplicationShim._getKeyCodeForChar(ch, KeyEvent.VK_UNDEFINED);
```

Comment:
Do you want to add a new test(s) for the cases when the hint is important?

Reply:
Looking through the Monocle code I see that it does support the numeric keypad and could benefit from the provided hint. I didn't find any bug in the database concerning KeyCharacterCombination support on Monocle so this may be a non-issue. If it is a problem I think it should be addressed in a separate PR at which point the test cases can be expanded.